TileFormats/VectorData/README.md

```diff
@@ -1,19 +1,248 @@
 
-Could also name this a `Geometry` tile.
+Each point, poyline, and polygon is a _feature_ in the 3D Tiles specification language. 
+
+## Layout
+
+A tile is composed of two sections: a header immediately followed by a body.
+
+**Figure1**: Vector tile layout.
+![](figures/layout.jpg)
+
+## Header
+
+The 44-byte header contains the following fields:
+
+| Field name | Data type | Description |
+| --- | --- | --- |
+| `magic` | 4-byte ANSI string | `"vctr"`. This can be used to identify the arraybuffer as a Vector tile. |
+| `version` | `uint32` | The version of the Vector Data format. It is currently `1`. |
+| `byteLength` | `uint32` | The length of the entire tile, including the header, in bytes. |
+| `featureTableJSONByteLength` | `uint32` | The length of the feature table JSON section in bytes. |
+| `featureTableBinaryByteLength` | `uint32` | The length of the feature table binary section in bytes. If `featureTableJSONByteLength` is zero, this will also be zero. |
+| `batchTableJSONByteLength` | `uint32` | The length of the batch table JSON section in bytes. Zero indicates that there is no batch table. |
+| `batchTableBinaryByteLength` | `uint32` | The length of the batch table binary section in bytes. If `batchTableJSONByteLength` is zero, this will also be zero. | 
+| `polygonIndicesByteLength` | `uint32` | The length of the polygon indices buffer. |
+| `polygonPositionsByteLength` | `uint32` | The length of the polygon positions buffer. |
+| `polylinePositionsByteLength` | `uint32` | The length of the polyline positions buffer. |
+| `pointPositionsByteLength` | `uint32` | The length of the point positions buffer. |
+
+If `featureTableJSONByteLength` equals zero, the tile does not need to be rendered.
+
+The body section immediately follows the header section, and is composed, in order, of four fields: `Feature Table`, `Batch Table`, `Indices`, `Positions`, and `Meshes`.
+
+Code for reading the header can be found in
+[Vector3DModelTileContent.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/vector-tiles/Source/Scene/Vector3DTileContent.js)
+in the Cesium implementation of 3D Tiles.
+
+## Feature Table
+
+Contains values for `vctr` semantics used to render features.  The general layout of a Feature Table is described in the [Feature Table specification](../FeatureTable).
+
+The `vctr` Feature Table JSON schema is defined in [vctr.featureTable.schema.json](../../schema/vctr.featureTable.schema.json).
+
+### Semantics
+
+Per-feature semantics specific to a feature type are prefixed with the name of the feature type. e.g. `POLYGON` for pre-triangulated polygons, `POLYLINE` for polylines and `POINT` for points.
+
+At least one global `LENGTH` semantic must be defined. 
+If `POLYGONS_LENGTH` is not defined, or zero, no polygons will be rendered. 
```

Comment:
Make each of these "If POLYGONS_LENGTH is not defined..." statements a bullet; otherwise, the "formatted" paragraph is hard to read.